Source/Document Structure/SvgSymbol.cs

```diff
@@ -1,10 +1,8 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
 using System.Drawing;
+using System.Drawing.Drawing2D;
 
-namespace Svg.Document_Structure
+namespace Svg.DocumentStructure
```

Comment:
Hm, I understand that this namespace fits better into the naming schema, but are you duplicating the whole class for this? I'm not sure that it is worth it...

Reply:
@mrbean-bremen 

`Svg.Document_Structure.SvgSymbol` will be removed.
So I added `Obsolete`.